index.html

```diff
@@ -239,13 +239,15 @@
         site, selecting a "Power Up" in an interactive game, or paying at a
         kiosk in a parking structure). The <a>PaymentRequest</a> allows
         developers to exchange information with the <a>user agent</a> while the
-        user is providing input before approving or denying a payment request.
+        user is providing input (up to the point of user approval or denial of
+        the payment request).
       </p>
       <p data-link-for="PaymentRequest">
-        The <a>user agent</a> as a whole has a single <dfn>payment request is
-        showing</dfn> boolean, initially false. This is used to prevent
-        multiple <a>PaymentRequest</a>s from being shown, via their
-        <a>show()</a> method, at the same time.
+        Because the simultaneous display of multiple <a>PaymentRequest</a> user
+        interfaces may confuse the user, this specification limits the <a>user
+        agent</a> to displaying one at a time via the <a>show()</a> method.
+        This is ensured by a <dfn>payment request is showing</dfn> boolean,
+        global to the browser and initially false.
```

Comment:
Please remove the ", global to the browser and initially false." Avoids duplication.